main.rb

```diff
@@ -31,6 +31,7 @@ def choose_option(input)
     when '8' then add_book
     when '9' then add_music_album
     when '10' then add_game
+    when '12' then add_author
```

Comment:
There is no such method "add_author". Please remove this option '12'

Reply:
Hi @SergSkol,
Thank you for notifying.
Removed the same.